tests/ui-api-layer-acceptance-tests/dex/dex.go

```diff
@@ -0,0 +1,34 @@
+package dex
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+func IsSCIEnabled() bool {
+	config := readConfigurationFile()
+	return strings.Contains(config, "SAP CI")
+}
+
+func readConfigurationFile() string {
+	path := os.Getenv("DEX_CONFIGURATION_FILE")
+	data, err := ioutil.ReadFile(path)
+	switch {
+	case os.IsNotExist(err):
+		return ""
+	default:
+		exitOnError(err, "while reading DEX configuration file")
+	}
+
+	return string(data)
+}
+
+func exitOnError(err error, context string) {
```

Comment:
I would like to see a different if statement:

func exitOnError(err error, context string) {
	if err == nil {
		return
	}
        log.Fatal(errors.Wrap(err, context))
}

also, do we need it at all, if we can inline it? 🙂